core/frontend/src/types/common.ts

```diff
@@ -0,0 +1,13 @@
+export interface Error {
+    index: ErrorIndex
+    message: string
+}
+
+export enum ErrorIndex {
+    WifiForgetFail = 'wifi_forget_fail',
```

Comment:
Why the strings are necessary for this enum ?

Reply:
They are not. Removing.